tests/Middleware/LazyLoadImagesTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace RenatoMarinho\LaravelPageSpeed\Test\Middleware;
+
+use RenatoMarinho\LaravelPageSpeed\Middleware\LazyLoadImages;
+use RenatoMarinho\LaravelPageSpeed\Test\TestCase;
+
+class LazyLoadImagesTest extends TestCase
+{
+    protected function getMiddleware()
+    {
+        $this->middleware = new LazyLoadImages();
+    }
+
+    public function testLazyLoadImages()
```

Comment:
@torosegon Thanks for contributing!!

The lazy loading regex is verifying if `loading` attribute doesn't exists. That's nice!

Can you add a test case for this situation?

Add an `<img>` with loading attribute `eager|auto` at [test page](https://github.com/renatomarinho/laravel-page-speed/blob/master/tests/Boilerplate/index.html) and make an assertion to ensure this behavior.